packages/round-manager/src/features/api/types.ts

```diff
@@ -0,0 +1,36 @@
+export interface Web3Instance {
+  /**
+   * Currently selected address in ETH format i.e 0x...
+   */
+  account: string;
+  /**
+   * Chain ID of the currently connected network
+   */
+  chainId: number;
+}
+
+export interface Program {
+  /**
+   * The on-chain unique program ID
+   */
+  id?: string;
+  /**
+   * The identifier which represents the program metadata on a decentralized storage
+   */
+  metadataIdentifier: string;
+  /**
+   * Addresses of wallets that will have admin privileges to operate the Grant program
+   */
+  operatorWallets: Array<string>;
+}
+
+export interface IPFSFile {
+  /**
+   * File content to be saved in IPFS
+   */
+  content: string;
+  /**
+   * Optional path
+   */
+  path?: string;
+}
```

Comment:
should we put each of these interfaces in it's own with within a folder called types ? cause we would have more types being created and wouldn't want this file to bloat up

Reply:
definitely in the plan for refactoring but i'd say it's left this way for simplicity atm until it's required